packages/forklift-console-plugin/src/modules/Providers/data.ts

```diff
@@ -11,6 +11,7 @@ import {
   IProvidersByType,
   IRHVProvider,
   IVMwareProvider,
+  IOpenStackProvider,
```

Comment:
run `npm run lint:fix`to autofix this lintin issue

Reply:
Done